pkg/core/native/native_neo.go

```diff
@@ -1187,9 +1187,19 @@ func (n *NEO) getAllCandidatesCall(ic *interop.Context, _ []stackitem.Item) stac
 	seekres := ic.DAO.SeekAsync(ctx, n.ID, storage.SeekRange{Prefix: prefix})
 	filteredRes := make(chan storage.KeyValue)
 	go func() {
+	loop:
 		for kv := range seekres {
+			select {
```

Comment:
Why this one? You can't avoid being in `keep()` when `ctx.Done()` happens.

Reply:
Transaction processing is already finished by this moment.

Reply:
OK, let's remove it, the code will be faster in case of full seekres traversal.